src/data_processing/post_processing/packing_characterization_run.py

```diff
@@ -84,49 +85,59 @@ def MeasureLocalPropertiesWithDifferentRadius(self):
         measured_mean_coordination_number = []
         measured_eigenvalues = []
         measured_second_invariant_of_deviatoric_tensor = []
+        measured_conductivity = []
 
         while side_length <= side_length_max:
 
             RVE_lambda_list.append(RVE_lambda)
             if self.parameters_DEMGen["packing_charcterization_setting"]["measure_density_option"]:
                 measured_packing_density.append(1 - self.MeasureSphereForGettingPackingProperties((side_length/2), center_x, center_y, center_z, 'porosity'))
-            
+
             if self.parameters_DEMGen["packing_charcterization_setting"]["measure_mean_coordination_number_option"]:
                 measured_mean_coordination_number.append(self.MeasureSphereForGettingPackingProperties((side_length/2), center_x, center_y, center_z, 'averaged_coordination_number'))
-            
+
             if self.parameters_DEMGen["packing_charcterization_setting"]["measure_anisotropy_option"]:
                 eigenvalues, second_invariant_of_deviatoric_tensor, measured_fabric_tensor = self.MeasureSphereForGettingPackingProperties((side_length/2), center_x, center_y, center_z, 'fabric_tensor')
```

Comment:
This should be updated, as you modified the output of the measurement function in Kratos